src/ILAMB/ConfCO2.py

```diff
@@ -11,6 +11,13 @@
 import pylab as plt
 import numpy as np
 import os
+import copy
+from ILAMB import ccgfilt
+import scipy
+import dateutil
+import csv
```

Comment:
I don't think you use this import, we want to be careful what we import as I have to make sure the user has it installed. Am I correct in noticing that it isn't used?